### ISSUE TITLE
Lambda crashes on fresh install

### ISSUE THREAD

**Issue by dwertheimer** [alexa/skill-sample-nodejs-college-finder#3]
I'm here at AWS:re-invent...decided to dig into your great/complex example. I can get it to run initially, but the lambda crashes eventually. Maybe not for you guys because you already cached your profile in the Dynamo. But for me, it dies like so:

>  User  >  alexa open college finder
>   Alexa >  Welcome back to U.S. College Finder. To get the best college matches, I'll need to know a little more about you. Would you like to finish your profile now?
>   User  >  yes
>   Alexa >  To make searching easier, let's build your student profile. You can ask me to skip any of these questions if you're not sure or don't have a preference. OK. First question. You can save your SAT score or ACT score. Which one do you have?
>   User  >  ACT
>   Alexa >  There was a problem with the requested skill's response

When I look at the logs, it seems to be dying because:

helpers.js: getSlotResolution(handlerInput, slot)
when getSlotResolution is called after user enters "ACT", slot is passed as: "SCORES" (but the name of the slot is actually "TEST" -- "SCORES" is the slotType)

```
handlerInput.requestEnvelope.request.intent.slots = 
{ TEST: 
{ name: 'TEST',
value: 'act',
resolutions: { resolutionsPerAuthority: [Array] },
confirmationStatus: 'NONE',
source: 'USER' } }
```

as you can see, intent.slots has a member called "TEST", but not "SCORES"
...so it eventually gets a TypeError (because ...slots[slot] is undefined

**Comment by jkelvie** [comment]
This issue is also resolved by this PR #5

**Comment by sharpstef** [comment]
Same issue as #4
